src/requests/wishlistRequests/addToWishList.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const addToWishList = (product, token) => {
+    return axios.post(
+        "/api/user/wishlist/",
+        { product },
+        { headers: { authorization: token } }
+    )
+};
+
+export { addToWishList };
```

Comment:
Won't be there too many files if you keep making different files for each function call? May I know the source of this inspiration?

Reply:
felt like I should oversimplify the code anyway it's in a different folder.

Reply:
Ok